### DISCUSSION TITLE
Documentation typo?

### DISCUSSION THREAD

**Discussion by schklom** [authelia/authelia#5269] [Q&A] [answered]
I noticed that in your documentation, `X-Forwarded-Ssl: on` is present for Nginx but not HAProxy.

https://www.authelia.com/integration/proxies/haproxy/#configuration-1 has no mention of the header, but https://www.authelia.com/integration/proxies/nginx/#proxyconf does.

Is this a typo, or is there a difference?

Traefik v2 does not forward this header by default.

**Comment by james-d-elliott** [comment, marked as answer]
The required headers for Authelia itself are listed here: https://www.authelia.com/integration/proxies/introduction/#required-headers

The required headers for Forwarded Authentication are listed here: https://www.authelia.com/integration/proxies/introduction/#destination-identification

The variance is just due to a common header that usually exists with the NGINX proxy.conf, probably due to some misc dated software from a similar era to NGINX.
